README.rst

```diff
@@ -229,6 +229,7 @@ We wish to thank the authors of `Alonso and Marder, 2019 <https://doi.org/10.755
 The part of the code in this repository developed by the EPFL Blue Brain Project was supported by funding to the Blue Brain Project, a research center of the École polytechnique fédérale de Lausanne (EPFL), from the Swiss government's ETH Board of the Swiss Federal Institutes of Technology.
 
 Copyright (c) 2023-2024 Blue Brain Project/EPFL
+Copyright (c) 2025 Open Brain Institute
```

Comment:
Minor: Could you add one more space before this line so that it appears in a new line?

Also, could you add some text in the "About" section on the top left of the repository's [main page](https://github.com/openbraininstitute/Currentscape)? Maybe: "Visualise ion currents & concentrations of electrical neuron models". 

Remember to also do it in [bluecellulab](https://github.com/openbraininstitute/BlueCelluLab).
Thanks!

Reply:
done :)